cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,82 @@
+/// <reference types="Cypress" />
+
+describe('EngageSphere', ()=> {
+  beforeEach(()=> {
+    cy.visit('/')
+  })
+  
+  context('Heading', ()=> {
+    it('Shows the heading and the theme button', ()=> {
+      cy.contains('h1','EngageSphere')
+        .should('be.visible')
+      cy.get('[id="theme-toggle-button"]')
+        .should('be.visible')
+    })
+  })
+
+  context('Greeting', ()=>{
+    it('Validate default greting', ()=>{
+      const pastDate = new Date(Date.UTC(1988, 3, 23))
+      cy.clock(pastDate)
+      cy.visit('/')
+      cy.contains('Hi there! It is now Fri Apr 22 1988')
+        .should('be.visible')
+    })
+
+    it('Validate personal greting', ()=>{
+      cy.get('[data-testid="name"]').type('Caio')
+      cy.contains('p','Hi Caio')
+        .should('be.visible')
+    })
+  })
+  context('Customer details modal', ()=>{
+    beforeEach(()=> {
+      cy.get('tbody tr').first().click()
+    })
+
+    it('goes back to the customer list when clicking the back button', ()=>{
+      cy.get('.customer-details')
+        .should('be.visible')
+      cy.contains('Back').click()
+      cy.get('[data-testid="table"]')
+        .should('be.visible')
+
+    })
```

Comment:
```suggestion
    })

```

Para melhorar a legibilidade, deixe uma linha em branco entre cada caso de teste (cada bloco `it`).

Reply:
O mesmo vale para outros casos de teste ao longo dos arquivos.